kraken-build/src/kraken/std/cargo/__init__.py

```diff
@@ -232,6 +237,42 @@ def cargo_deny(
     )
 
 
+def cargo_hack(
+    *,
+    project: Project | None = None,
+    features: CargoHackFeatures = CargoHackFeatures.EACH,
```

Comment:
For usability within `.kraken.py` it would be useful if this also accepted a string! So I can use `cargo_hack(features="powerset")` which is easier to type, read and remember. In case you didn't know, `Enum` types have a `__getitem__` support so that's straight forward to implement :)

Reply:
Done — I've made it accept either types and used `isinstance()` to figure out the type at runtime, to convert it. Hope that works :)